app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/PointFeatureTypeStrategy/PointFeatureTypeStrategy.ts

```diff
@@ -3,23 +3,26 @@ import {FeatureType} from "../../api/graphqlGlobalTypes";
 import {FeatureAttributeName} from "../../states/map/FeatureAttributeName";
 import {Dispatch} from "redux";
 import {FeaturesByType} from "../../states/map/FeaturesByType";
+import {MapFeatureTypeState} from "../../states/map/MapFeatureTypeState";
 
 export abstract class PointFeatureTypeStrategy implements FeatureTypeStrategy {
   getClickedQuery(clickedFeatureUri: string): null {
     return null;
   }
-  abstract dispatchLayerConfigurationActions(
-    keplerLayerOfFeatureType: any,
-    keplerFiltersOfFeatureType: any,
-    keplerFieldsOfFeatureType: any,
-    keplerInteractionConfigOfFeatureType: any,
-    dispatch: Dispatch<any>,
+  abstract dispatchLayerConfigurationActions(kwds: {
```

Comment:
Not sure what's going on here. You declared the parameter object interface but aren't using it? If there are optional members then add them as ? to the interface, or have another interface that extends the first one.

Reply:
Oh big mistake. I created the interface after and I forgot to replace the parameter object